book/src/example.md

```diff
@@ -1,138 +1,13 @@
-> ```
-> serviceaccount/csi-snapshotter created
-> clusterrole.rbac.authorization.k8s.io/external-snapshotter-runner created
-> clusterrolebinding.rbac.authorization.k8s.io/csi-snapshotter-role created
-> ```
-
-### Deploy driver-registrar and hostpath CSI plugin in DaemonSet pod
-
-The CSI sidecar apps are going to connect to the CSI driver, therefore
-starting it first helps avoid timeouts and intermittent container
-restarts:
-
-> $ kubectl create -f [https://raw.githubusercontent.com/kubernetes/kubernetes/f40a5d1155aae95105a4e9bb8933d750c666e350/test/e2e/testing-manifests/storage-csi/hostpath/hostpath/csi-hostpathplugin.yaml](https://github.com/kubernetes/kubernetes/blob/f40a5d1155aae95105a4e9bb8933d750c666e350/test/e2e/testing-manifests/storage-csi/hostpath/hostpath/csi-hostpathplugin.yaml)
-> `daemonset.apps/csi-hostpathplugin created`
->
-> $ kubectl get pod
-> ```
-> NAME                       READY   STATUS    RESTARTS   AGE
-> csi-hostpathplugin-4k7hk   2/2     Running   0          22s
-> ```
-
-### Deploy CSI provisioner in StatefulSet pod
-
-> $ kubectl create -f [https://raw.githubusercontent.com/kubernetes/kubernetes/f40a5d1155aae95105a4e9bb8933d750c666e350/test/e2e/testing-manifests/storage-csi/hostpath/hostpath/csi-hostpath-provisioner.yaml](https://github.com/kubernetes/kubernetes/blob/f40a5d1155aae95105a4e9bb8933d750c666e350/test/e2e/testing-manifests/storage-csi/hostpath/hostpath/csi-hostpath-provisioner.yaml)
-> ```
-> service/csi-hostpath-provisioner created
-> statefulset.apps/csi-hostpath-provisioner created
-> ```
->
-> $ kubectl get pod
-> ```
-> NAME                         READY   STATUS    RESTARTS   AGE
-> csi-hostpath-provisioner-0   1/1     Running   0          14s
-> csi-hostpathplugin-4k7hk     2/2     Running   0          75s
-> ```
-
-
-### Deploy CSI attacher in StatefulSet pod
-
-> $ kubectl create -f [https://raw.githubusercontent.com/kubernetes/kubernetes/f40a5d1155aae95105a4e9bb8933d750c666e350/test/e2e/testing-manifests/storage-csi/hostpath/hostpath/csi-hostpath-attacher.yaml](https://github.com/kubernetes/kubernetes/blob/f40a5d1155aae95105a4e9bb8933d750c666e350/test/e2e/testing-manifests/storage-csi/hostpath/hostpath/csi-hostpath-attacher.yaml)
-> ```
-> service/csi-hostpath-attacher created
-> statefulset.apps/csi-hostpath-attacher created
-> ```
->
-> $ kubectl get pod
-> ```
-> NAME                         READY   STATUS    RESTARTS   AGE
-> csi-hostpath-attacher-0      1/1     Running   0          14s
-> csi-hostpath-provisioner-0   1/1     Running   0          56s
-> csi-hostpathplugin-4k7hk     2/2     Running   0          117s
-> ```
+The CSI sidecar apps are going to connect to the CSI driver, therefore starting it first helps avoid timeouts and intermittent container restarts
```

Comment:
I'm not sure what this sentence means and how it translates into something you can do in your Pod specs. Can you clarify?

Reply:
Put the CSI driver container at the top of the list of containers in the pod, if that pod also contains the sidecards.

It's less of an issue now because we improved the connection behavior of the sidecars, but previously we've seen sidecars time out in their connection attempt because the CSI driver was started after them and image pulling took too long. Having said that, I still think it's a good advice.

Reply:
This is an advice about best-practices. Thinking about it some more, I now suggest that we remove the entire sentence. It doesn't belong into the example page.

Reply:
I deleted that sentence and pushed new state